src/amo/utils.js

```diff
@@ -1,10 +1,36 @@
-import { LANDING_PAGE_ADDON_COUNT } from 'amo/constants';
+import {
+  FEATURED_ADDONS_TO_LOAD,
+  LANDING_PAGE_ADDON_COUNT,
+} from 'amo/constants';
+import { getFeatured, loadFeatured } from 'amo/actions/featured';
 import { getLanding, loadLanding, failLanding } from 'amo/actions/landing';
 import { featured as featuredAPI, search } from 'core/api';
 import { SEARCH_SORT_POPULAR, SEARCH_SORT_TOP_RATED } from 'core/constants';
 import { apiAddonType } from 'core/utils';
 
 
+export function fetchFeaturedAddons({ addonType, api, dispatch }) {
```

Comment:
I guess it can be part of it. This was just convention in previous similar utils functions (categories and search) and the smaller functions make it easier to test and such. I like the small functions personally, but if you'd rather it all together I can put them together.

Though since I'm not discretely testing it as you mentioned, I'll either do that or merge them; no use doing it halfway!